manifest-build-tools/post_test.sh

```diff
@@ -0,0 +1,217 @@
+# --boxFile ./someDir/some.box
+# --controlNetwork vmnet*
+#
+# docker-post-test need some special parameter of docker build record file and cloned RackHD repo
+# --RackHDDir ./someDir/RackHD
+# --buildRecord ./record_file
+#
+############################################
+
+set -x
+
+while [ "$1" != "" ];do
+    case $1 in
+        --type)
+            shift
+            type=$1;;
+        --adminIP)
+            shift
+            adminIP=$1;;
+        --adminGateway)
+            shift
+            adminGateway=$1;;
+        --adminNetmask)
+            shift
+            adminNetmask=$1;;
+        --adminDNS)
+            shift
+            adminDNS=$1;;
+        --datastore)
+            shift
+            datastore=$1;;
+        --deployName)
+            shift
+            deployName=$1;;
+        --ovaFile)
+            shift
+            ovaFile=$1;;
+        --vcenterHost)
+            shift
+            vcenterHost=$1;;
+        --ntName)
+            shift
+            ntName=$1;;
+        --ntPass)
+            shift
+            ntPass=$1;;
+        --esxiHost)
+            shift
+            esxiHost=$1;;
+        --boxFile)
+            shift
+            boxFile=$1;;
+        --controlNetwork)
+            shift
+            controlNetwork=$1;;
+        --RackHDDir)
+            shift
+            RackHDDir=$1;;
+        --buildRecord)
+            shift
+            buildRecord=$1;;
+        *)
+        exit 1
+    esac
+    shift
+done
+
+findRackHDService() {
+    case $type in
+        ova)
+        api_test_result=`ansible ova-for-post-test -a "wget --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 -t 1 --continue localhost:8080/api/2.0/nodes"`
+        echo $api_test_result | grep "$service_normal_sentence" > /dev/null  2>&1
+        ;;
+        docker)
+        wget --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 -t 1 --continue http://172.31.128.1:9080/api/2.0/nodes
+        ;;
+        vagrant)
+        curl 127.0.0.1:9090/api/2.0/nodes
+        ;;
+    esac
+}
+
+waitForAPI() {
+    #will be edit after build own ova
+    service_normal_sentence="Authentication Failed"
+    timeout=0
+    maxto=60
+    while [ ${timeout} != ${maxto} ]; do
+        findRackHDService
+        if [ $? = 0 ]; then
+          echo "RackHD services perform normally!"
+          break
+        fi
+        sleep 10
+        timeout=`expr ${timeout} + 1`
+    done
+    if [ ${timeout} == ${maxto} ]; then
+        echo "Timed out waiting for RackHD API service (duration=`expr $maxto \* 10`s)."
+        exit 1
+      fi
+}
+
+############################################
+# ova post test
+############################################
+
+deploy_ova() {
+    echo yes | ovftool \
+    --prop:adminIP=$adminIP  --prop:adminGateway=$adminGateway --prop:adminNetmask=$adminNetmask  --prop:adminDNS=$adminDNS \
+    --overwrite --powerOffTarget --powerOn --skipManifestCheck \
+    --net:"ADMIN"="External Connection" \
```

Comment:
Why not specific network for eth1 ?
and moreover, the vSwitch name "External Connection" should be a variable of script , because it varies from Jenkins set up .